core/core-impl/src/main/java/ai/timefold/solver/core/impl/localsearch/decider/LocalSearchDecider.java

```diff
@@ -94,15 +94,9 @@ public void decideNextStep(LocalSearchStepScope<Solution_> stepScope) {
         for (Move<Solution_> move : moveSelector) {
             LocalSearchMoveScope<Solution_> moveScope = new LocalSearchMoveScope<>(stepScope, moveIndex, move);
             moveIndex++;
-            // TODO use Selector filtering to filter out not doable moves
```

Comment:
Geoffrey obviously expected something like this PR would eventually get done.
I'm assuming it never got done because of the size issues above.